src/summary/history/notes-card.component.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { match } from "react-router";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import HorizontalLabelValue from "../cards/horizontal-label-value.component";
+import { encounterResource } from "./encounter.resource";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+import dayjs from "dayjs";
+
+export default function NotesCard(props: NotesCardProps) {
+  const [patientNotes, setPatientNotes] = React.useState(null);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+
+    encounterResource(props.currentPatient.identifier[0].value, abortController)
+      .then(notes => setPatientNotes(notes))
+      .catch(error => {
+        setTimeout(() => {
+          throw error;
+        });
+      });
+
+    return () => abortController.abort();
+  }, [props.currentPatient.identifier[0].value]);
+
+  return (
+    <SummaryCard name="Notes" match={props.match} styles={{ width: "45.5rem" }}>
+      <SummaryCardRow>
+        <SummaryCardRowContent justifyContent="space-between">
+          <HorizontalLabelValue

```

Comment:
In my opinion, we shouldn't use the HorizontalLabelValue component for the Notes header. The HorizontalLabelValue component is made for a situation where there are two values, but you need three. Additionally, "Date" is not a label for "Encounter type".

You should just write your own jsx for the header.

Reply:
I have an idea in mind, what of having both label and value optional so as to generate 3 or multiple values

Reply:
No. The HorizontalLabelValue component doesn't make sense here (it is not a header). We should change this to not use HorizontalLabelValue for the Notes header.